### PR TITLE
added CODEOWNERS file

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
I enabled that pull requests require a review of the code owner before they can be merged. This file tells GitHub who owns what.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3057/pr-test (or any other room on that server)

**Review by 96LawDawg — DISMISSED** [review]
This will be a little more work and will slow things down, but that is probably a good thing.

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# UI
+*.css @RaphaelAlvez
+*.html @RaphaelAlvez
+*.svg @RaphaelAlvez
+
+# Library
+library @96LawDawg
+
+# Tests
+/tests/ @UltimateGeek
```
Comment (UltimateGeek) [inline CODEOWNERS:13]:
```suggestion
jest.config.mjs @UltimateGeek
/tests/ @UltimateGeek
```

**Review by UltimateGeek — CHANGES_REQUESTED** [review]

**Review by UltimateGeek — APPROVED** [review]
Thank you for establishing this. :-)

**Review by 96LawDawg — APPROVED** [review]
Looks good.  I approve.

**Pull request merged**